plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeConnectorSmokeTest.java

```diff
@@ -87,6 +92,7 @@
 import static java.util.concurrent.Executors.newFixedThreadPool;
```

Comment:
Note to self - add in the commit comment.

```
Relevant example taken from Databricks documentation in regards to the
distinction between `WriteSerializable` and `Serializable` isolation levels:

> For example, consider `txn1`, a long running delete and `txn2`,
> which inserts blindly (that is, operations that blindly append data
. without reading any data) data deleted by `txn1`.
> `txn2` and `txn1` complete and they are recorded in the order
> `txn2, txn1`
> into the history of the table.
> According to the history, the data inserted in `txn2` should not exist
> in the table. For `Serializable` level, a reader would never see data
> inserted by `txn2`. However, for the `WriteSerializable` level, a reader
> could at some point see the data inserted by `txn2`.

```

Reply:
add as well https://github.com/delta-io/delta/blob/master/PROTOCOL.md#Action-Reconciliation